lib/core/MultiLineFishboneLayoutStrategy.ts

```diff
@@ -100,8 +100,9 @@ class SingleFishboneLayoutAdapter extends LayoutStrategyBase {
     this.SpecialRoot = new TreeNodeView(
       Box.Special(Box.None, realRoot.Element.Id, true)
     );
-    (this.SpecialRoot.Level = this.RealRoot.Level),
-      (this.SpecialRoot.ParentNode = this.RealRoot);
+
```

Comment:
As far as I can see, this syntax does practically the same; thus I had to change it as Typescript 4.5.0-dev was not capable to recognise the syntax.